README.md

```diff
@@ -230,6 +232,11 @@ function (/*req, res*/) {
 }
 ```
 
+### reqRateLimitParam
```

Comment:
Let's rename this to `reqestPropertyName`. I feel like that's a little bit more clear as to what this controls.

Reply:
Yeah it's ok.